app/controllers/concerns/controller_helper.rb

```diff
@@ -0,0 +1,21 @@
+module Concerns::ControllerHelper
```

Comment:
I remember I wrote a comment about renaming this controller concern as ViewHelper, but I can't find it :shrug:  

The goal is to bring the `#view_helper` method, so let's rename the module, and make the other methods `private` to the module (so they won't get mixed into the controllers):

```ruby
module Concerns::ViewHelper
  extend ActiveSupport::Concern

  def view_helper(...)
  end

  private

  def date_format
  end

  ...
end
```